sketch_map_tool/analyses/modules/analysis_sources.py

```diff
@@ -24,25 +24,48 @@ class SourcesAnalysis(Analysis):
     """
     Check if there are sources accountable for a substantial share of all features
     """
+
     importance = 0  # Manual inspection needed to estimate the trustworthiness
     threshold_yellow = 0.02  # i.e. 2 %
-    threshold_red = None  # A red level cannot be reached as manual inspection is necessary
+    threshold_red = (
+        None  # A red level cannot be reached as manual inspection is necessary
+    )
 
     title = "Source Analysis"
     plot_name = "_plot_sources.png"
 
-    white_list = ("BAG", "Bing", "bing", "digitalglobe", "Mapbox", "cuzk:km", "Fugro 2005",
-                  "landsat", "uhul:ortofoto", "yahoo", "AGIV", "aerial imagery", "survey", "image",
-                  "mapillary", "local knowledge", "GPS", "common knowledge", "geoimage.at",
-                  "HiRes aerial imagery")  # Typical sources that are either known for sufficient
+    white_list = (
+        "BAG",
+        "Bing",
+        "bing",
+        "digitalglobe",
+        "Mapbox",
+        "cuzk:km",
+        "Fugro 2005",
+        "landsat",
+        "uhul:ortofoto",
+        "yahoo",
+        "AGIV",
+        "aerial imagery",
+        "survey",
+        "image",
+        "mapillary",
+        "local knowledge",
+        "GPS",
+        "common knowledge",
+        "geoimage.at",
+        "HiRes aerial imagery",
+    )  # Typical sources that are either known for sufficient
     #                                        quality or where inspection is impossible. These
```

Comment:
These comments are over indented after the change

Reply:
Fixed